tests/integration/api/test_get_edition.py

```diff
@@ -1,10 +1,20 @@
 import pytest
 import requests
+import json
 from .constants import API_URL
 from .utils import assert_response_status
 
+@pytest.fixture(scope="module", autouse=True)
+def seed_test_data():
+    process = SeedTestDataProcess()
+    process.runProcess()
```

Comment:
I think we can use seed the data directly from fixtures. Let's move this to conftest.py.